applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -354,6 +354,12 @@ config BT_AUDIO_BROADCAST_ID_FIXED
 	  Fixed broadcast ID; 3 octets. Will only be used if BT_AUDIO_USE_BROADCAST_ID_RANDOM=n.
 	  Should only be used for debugging.
 
+config BT_AUDIO_INTERLEAVED
+	bool "Interleaved packing"
+	default n
+	help
+	  ISO channels can either be interleaved or sequentially packed; default is sequential.
```

Comment:
Nitpick:
I also used ";" in doc/text before. But I remember it was pointed out to me that we should try to avoid using ";"

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/9829#discussion_r1063248266

Reply:
Then the master has spoken